nodejs/eks/cluster.ts

```diff
@@ -254,12 +270,43 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         kubeconfig: kubeconfig,
         provider: provider,
         vpcCni: vpcCni,
-        instanceRole: instanceRole,
-        instanceProfile: instanceProfile,
+        instanceProfile: instanceProfile! || undefined,
         eksNodeAccess: eksNodeAccess,
     };
 }
 
+// Add the specified policies to the existing IAM role.
+export type Policies =  pulumi.Input<aws.ARN>[];
+export function addPoliciesToExistingRole(
+    name: string,
+    role: aws.iam.Role,
+    policies: Policies) {
+
+    let counter = 0;
+    for (const policy of policies) {
+        // Create RolePolicyAttachment without returning it.
+        const rpa = new aws.iam.RolePolicyAttachment(
+            `${name}-policy-${counter++}`,
+            {
+                policyArn: policy,
+                role: role,
+            },
+        );
+    }
+}
```

Comment:
Not sure we should include this.  It's very general - not something related to EKS at all.  If we were to add a helper like this - we could do so in the AWS pacakge.